src/domain/Chat/Chat.module.scss

```
@@ -86,55 +113,61 @@
       }
     }
 
+    @keyframes fadeIn {
```

Stylelint (17.2.0)

[error] Expected keyframe name "fadeIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)